README.md

```diff
@@ -29,6 +33,14 @@ curl -sSL https://raw.githubusercontent.com/soywod/himalaya/master/install.sh |
 curl -sSL https://raw.githubusercontent.com/soywod/himalaya/master/install.sh | PREFIX=~/.local sh
 ```
 
+### Install via brew
```

Comment:
Instructions add to the wiki at https://github.com/soywod/himalaya/wiki/Installation:using-brew. Could you remove this from the PR?

Reply:
Sounds good, actually this is the first project that I saw using wiki so seriously 😄

Reply:
Just did the change. 👍